src/domain/shared/primitives.ts

```diff
@@ -26,6 +26,18 @@ export const ZERO_BANK_FEE = {
   btcBankFee: ZERO_SATS,
 }
 
+// 100_000 BTC is the max amount you can make a hodl invoice for in lnd
+export const MAX_SATS = {
+  currency: WalletCurrency.Btc,
+  amount: 10_000_000_000_000n,
+} as const
+
+// Assumes optimistic price under sat-cent parity
+export const MAX_CENTS = {
+  currency: WalletCurrency.Usd,
+  amount: 10_000_000_000_000n,
```

Comment:
I just realized, this should probably be based on a pessimistic price to function properly as a max that doesn't trigger lnd errors when it gets converted to sats. Maybe something like `10_000_000_000` which allows you to do less effective sats after conversion, but works as expected for prices down to $1k/BTC.

Does it make sense to have this max here, or should I add checks individually in certain places (like `registerInvoice` method) to check amounts after conversion from USD?

Reply:
a nit: for cents you may want to put the `_` after two `00`? ie: `const hundredk = 100_000_00`

Reply:
Oh yes, it's 100B actually. I fixed the format. I left the number as is to be closer to the value we're defining for `MAX_SATS` since this is more for data type validation than business logic https://github.com/GaloyMoney/galoy/pull/1576/commits/e00ef5e1710f0aa5d9973b7162cd20802ef06edd